src/d3d9/d3d9_interfaces.h

```diff
@@ -28,6 +28,17 @@ ID3D9VkInteropInterface : public IUnknown {
   virtual void STDMETHODCALLTYPE GetPhysicalDeviceHandle(
           UINT                  Adapter,
           VkPhysicalDevice*     pPhysicalDevice) = 0;
+
+  /**
+   * \brief Gets a list of enabled instance extensions
+   * 
+   * \param [out] pExtensionCount Number of extensions
+   * \param [out] ppExtensions List of extension names
+   * \returns D3DERR_MOREDATA if the list was truncated
+   */
+  virtual HRESULT STDMETHODCALLTYPE GetInstanceExtensions(
```

Comment:
This needs to be moved to `ID3D9VkInteropInterface1` with its own GUID, otherwise anything using this will explode violently.

Reply:
Done. When this merges I'll move the stuff in #4506 to that interface too.